Clear for a single user throws an error

When you try to clear a user's message with <code>\clear -u [@user]</code> the bot does not respond and throws the following error in the terminal: <code>[ERROR] [tried to log empty message]</code>

--- a/src/commands/CommandClear.java
+++ b/src/commands/CommandClear.java
@@ -228,7 +228,7 @@ public class CommandClear extends BotCommand {
 					false, "i", "interactive"),
 			new ParametersHelp(
 				"Allows you to delete the messages of a user you specifie.",
-				false, "u", "user"),
+				true, "u", "user"),
 			new ParametersHelp(
 				"Allows you to delete your own messages.",
 				false, "s", "self"),

